db/migrate/20180919152423_create_quote_types.rb

```diff
@@ -0,0 +1,11 @@
+class CreateQuoteTypes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :quote_types do |t|
+      t.string :name, limit: 255, null: false
+      t.string :order, limit: 255, null: false
```

Comment:
What is this field for?

Reply:
:order is used to order the QuoteTypes, example "'initial' code 1, 'in_analisis' code 2 ..."

Reply:
@vinibr3 this field name `order` doesn't make sense in this context, what you want is `state`